user-interface/src/main/java/life/qbic/datamanager/views/projects/project/samples/BatchDetailsComponent.java

```diff
@@ -276,15 +277,21 @@ public static class DeleteBatchEvent extends ComponentEvent<BatchDetailsComponen
     @Serial
     private static final long serialVersionUID = -5424056755722207848L;
     private final BatchId batchId;
+    private final String batchLabel;
 
     public DeleteBatchEvent(BatchDetailsComponent source, BatchId batchId,
-        boolean fromClient) {
+        String batchLabel, boolean fromClient) {
```

Comment:
why did this change in this PR?

Reply:
Because the notification now also can contain the batch label